File: frontend/src/DTO/MemberDTO/MemberDTO.ts
interface MemberDTO {
    jwt: string;
    groupId: number;
}

interface MemberRoleDTO {
    jwt: string;
    userId: number;
    roleId: number;
    groupId: number;
}

interface MemberItem {
    id: number;
    name: string;
    role: string;
}

export { MemberDTO, MemberRoleDTO, MemberItem };

File: backend/DTO/Group/MemberDTO.cs
public class MemberDTO
{
    public string Jwt { get; set; }
    public int UserId { get; set; }
    public int GroupId { get; set; }
}

public class MemberItemDTO
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Role { get; set; }
}